apps/dash-datashader/app.py

```diff
@@ -0,0 +1,316 @@
+# coding: utf-8
+
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output, State
+import plotly.plotly as py
+import plotly.graph_objs as go
+import datashader as ds
+import datashader.transfer_functions as tf
+import pandas as pd
+import numpy as np
+import json
+import copy
+import xarray as xr
+from collections import OrderedDict
+
+#######################################################################################################################
```

Comment:
Please get rid of these long comment lines. You can keep the one below that just says `# Data generation`, though :)